en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+   - Bit 4 for `PARACHUTE_TRIGGER_FLAGS_GEOFENCE` is set to "0" to disarm.
+- The second parameter would be `0b11001`.
+   - Since bit 0, 1, 2, and 4 are being configured and no other trigger sources are being configured, all of those bits are set to "1" to indicate that those specific trigger sources need their arm statuses to be updated.
+   - Although bit 3 in the first parameter is set to "0", since bit 3 is set to "0" in the second parameter, the parachute should ignore the bit in the first parameter and not change the arm/disarm status of the trigger source mapped to bit 3.
+
+
+##### PARACHUTE_TRIGGER_FLAGS
+
+[PARACHUTE_TRIGGER_FLAGS](#PARACHUTE_TRIGGER_FLAGS) is a bitmask that is used in the parameters of the [MAV_CMD_SET_PARACHUTE_ARM](#MAV_CMD_SET_PARACHUTE_ARM) command and is published in the [PARACHUTE_STATUS](#PARACHUTE_STATUS) message. When published in the [PARACHUTE_STATUS](#PARACHUTE_STATUS) message, the bitmask details the various arm/disarm states of all the trigger flags.
+
+A list of the various flags can be found below:
+
+| Bit| Value  | Parameter                                      | Description                                                                                                            |
+|----|--------|------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| 0  | 1      | `PARACHUTE_TRIGGER_FLAGS_MANUAL`              | Manual trigger (ground based control via parachute-specific RF channel)                                                |
+| 1  | 2      | `PARACHUTE_TRIGGER_FLAGS_ATS`                 | Automatic trigger system (ATS)                                                                                         |
+| 2  | 4      | `PARACHUTE_TRIGGER_FLAGS_FC`                  | Flight controller trigger (e.g. MAVLink from FC, PWM, DroneCan, RC Control)                                            |
+| 3  | 8      | `PARACHUTE_TRIGGER_FLAGS_OFFBOARD`            | Offboard computer trigger (via MAVLink)                                                                                |
+| 4  | 16     | `PARACHUTE_TRIGGER_FLAGS_GEOFENCE`            | Geofence trigger (by parachute). Parachute uses MAVLink mission protocol to fetch geofence.                            |
+| 5  | 32     | `PARACHUTE_TRIGGER_FLAGS_FTS_PRECHECKING`     | FTS (flight termination system) pre-checking protocol trigger                                                          |
+| 6  | 64     | `PARACHUTE_TRIGGER_FLAGS_ATS_AUTO_ARM`        | Auto-arming of parachute automatic trigger system (ATS). This allows a parachute to enable ATS after reaching a desired altitude. |
+| 7  | 128    | `PARACHUTE_TRIGGER_FLAGS_ATS_AUTO_DISARM`     | Auto-disarming of parachute automatic trigger system (ATS). This allows a parachute to disable ATS after detecting that it is below a desired altitude. |
```

Comment:
I thought auto-disarm is not altitude based, but stationary upon landing based?

Reply:
dependent on the developer. I think this can be kept as is to mirror the PR.

Reply:
we should then specify that the implementation of the auto arm/disarm conditions is developer-specific